claws/openclaw/sageox-summary/scripts/update-ox.sh

```
@@ -0,0 +1,113 @@
+  *)
+    echo "warning: clone_path not under \$HOME; skipping auto-update: $CLONE_PATH" >&2
+    exit 0
+    ;;
+esac
+case "$CLONE_PATH" in
+  *..*)
+    echo "warning: clone_path contains '..'; skipping auto-update: $CLONE_PATH" >&2
+    exit 0
+    ;;
+esac
+for ch in ';' '$' '`' '|' '&' '<' '>' '(' ')' '{' '}' '*' '?' '[' ']' '!' '\'; do
```

Shellcheck (0.11.0)

[info] Want to escape a single quote? echo 'This is how it'\''s done'.

(SC1003)